packages/gatsby-recipes/recipes/gatsby-theme-blog-core.mdx

```diff
@@ -0,0 +1,32 @@
+# Gatsby Theme Blog Core Starter Recipe
+
+[Gatsby theme blog core](https://www.gatsbyjs.org/packages/gatsby-theme-blog-core/) is a great theme for adding blog functionality to your site. It's designed to handle sourcing your blog data for you so you can style it however you like.
+
+---
+
+Install necessary NPM packages
+
+<NPMPackage name="gatsby-theme-blog-core" />
+
+---
+
+Install the gatsby-theme-blog-core plugin in gatsby-config.js. Make use of the `basePath` configuration so the blog listing appears at `/blog`.
+
+<GatsbyPlugin name="gatsby-theme-blog-core" options={{basePath: `/blog`}}/>
+
+---
+Now let's add a post! (this part isn't working)
```

Comment:
what's not working here?

Reply:
This is what happens when I move between versions. That's old and outdated :D